posthog/tasks/usage_report.py

```diff
@@ -489,6 +512,10 @@ def send_all_org_usage_reports(
         # teams_with_event_count_by_name=get_teams_with_event_count_by_name(period_start, period_end),
         teams_with_recording_count_in_period=get_teams_with_recording_count_in_period(period_start, period_end),
         teams_with_recording_count_total=get_teams_with_recording_count_total(),
+        teams_with_decide_requests_in_period=get_teams_with_decide_requests_count_in_period(period_start, period_end),
```

Comment:
Q: @raquelmsmith do you need total counts as well?

Reply:
Oh, people _may_ acutally use the `count_total` values in insights somewhere... but no need to add it here for now, we can always add later if we find that we want it for some reason.